handler.go

```diff
@@ -188,99 +188,106 @@ func (ca *Cagent) PostResultToHub(ctx context.Context, result *Result) error {
 	return nil
 }
 
-func (ca *Cagent) GetAllMeasurements() (common.MeasurementsMap, error) {
+func (ca *Cagent) CollectMeasurements(full bool) (common.MeasurementsMap, error) {
 	var errCollector = common.ErrorCollector{}
 	var measurements = make(common.MeasurementsMap)
 
 	cpum, err := ca.CPUWatcher().Results()
 	errCollector.Add(err)
 	measurements = measurements.AddWithPrefix("cpu.", cpum)
 
-	info, err := ca.HostInfoResults()
-	errCollector.Add(err)
-	measurements = measurements.AddWithPrefix("system.", info)
-
-	ipResults, err := IPAddresses()
-	errCollector.Add(err)
-	measurements = measurements.AddWithPrefix("system.", ipResults)
-
 	fsResults, err := ca.FSWatcher().Results()
 	errCollector.Add(err)
 	measurements = measurements.AddWithPrefix("fs.", fsResults)
 
-	netResults, err := ca.NetWatcher().Results()
-	errCollector.Add(err)
-	measurements = measurements.AddWithPrefix("net.", netResults)
-
 	mem, memStat, err := ca.MemResults()
 	errCollector.Add(err)
 	measurements = measurements.AddWithPrefix("mem.", mem)
 
-	proc, processList, err := ca.ProcessesResult(memStat)
-	errCollector.Add(err)
-	measurements = measurements.AddWithPrefix("proc.", proc)
-
-	ports, err := ca.PortsResult(processList)
-	errCollector.Add(err)
-	measurements = measurements.AddWithPrefix("listeningports.", ports)
-
-	swap, err := ca.SwapResults()
+	cpuUtilisationAnalysisResult, cpuUtilisationAnalysisIsActive, err := ca.CPUUtilisationAnalyser().Results()
 	errCollector.Add(err)
-	measurements = measurements.AddWithPrefix("swap.", swap)
+	measurements = measurements.AddWithPrefix("cpu_utilisation_analysis.", cpuUtilisationAnalysisResult)
+	if cpuUtilisationAnalysisIsActive {
+		measurements = measurements.AddWithPrefix(
+			"cpu_utilisation_analysis.",
```

Comment:
This is all code that was sightly moved up. 
Can't fix spelling as it will break compatibility with backend.

GolangCI checks are temporarely disabled.